db/migrate/20250128141033_create_feedbacks.rb

```diff
@@ -0,0 +1,10 @@
+class CreateFeedbacks < ActiveRecord::Migration[7.2]
```

Comment:
Doing this is OK for testing but we don't want to commit this as it may not be necessary. If you create an ActiveModel object then it doesn't need a database migration.